code/style.css

```diff
@@ -1,4 +1,231 @@
 html {
-  background: #0025ff;
-  color: #fff;
+  background: #f7f7fa;
+  color: rgb(8, 8, 8);
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
 }
+
+.container.grid {
+  padding: 2 rem; 
+  display: grid;
+  grid-template-columns: 2fr 1fr 1fr;
+  grid-gap: 1rem;
+}
+
+.title{
+  padding: 1 rem;
+}
+
+.cell {
+  padding: 1 rem;
+}
+
+a.ex1:hover, a.ex1:active {
+  font-size: 150%;
+}
+
+.meny {
+  font-size: x-small;
+  padding: 1 rem;
+  color: rgb(31, 93, 95);
+  align-self: center;
+}
+
+.news-img {
+  width: 100%;
+  margin-top: 30px;
+}
+
+img {
+  height: 300px;
+  width: 300px;
+}
+
+.undertitle {
+  font-style: italic;
+  font-size: small;
+ } 
+
+ button {
+  height: 30px;
+  border: 1px solid;
+  border-color: rgb(31, 93, 95);
+  color: rgb(31, 93, 95);
+  background-color: #f7f7fa;
+  font-size: 12px;
+  margin-left: 140px;
+  justify-content: center;
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
+}
+
+.gallery {
+  margin-top: 50px;
+  color: rgb(31, 93, 95);
+  margin-left: 140px;
+}
+
+.grid-container {
+  display: grid;
+  grid-template-columns: 1fr; 
+  gap: 5px; 
+  margin-top: 30px;
+  margin-bottom: 20px;
+}
+
+img {
+  width: 100%;
+}
+
+.bottom-grid-container {
+  display: grid;
+  border: transparent;
+  height: auto;
+  width: auto;
+  grid-template-columns: 1fr; 
+  gap: 5px; 
+  margin-top: 50px;
+  justify-content: center;
+  align-items: center;
+  align-self: center;
+}
+
+.grid-item9, .grid-item10, .grid-item11, .grid-item12, .grid-item13, .grid-item14 {
+border: 1px solid;
+border-color: rgb(31, 93, 95);
+align-self: center;
+}
+
+img {
+  width: 100%;
+}
+
+footer {
+  margin-top: 30px;
+  text-align: center;
+  border-top: 1px solid;
+  border-color: rgb(31, 93, 95);
+  padding: 40px;
+  font-size: small;
+  color: rgb(31, 93, 95);
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
+}
+
+@media (min-width: 668px) {
```

Comment:
The use of media query is well understood and the measures perfect, but I recommend using only one media query per size and add all the elements inside to avoid repetition.